spec/aptible/cli/helpers/tunnel_spec.rb

```diff
@@ -35,13 +35,13 @@
   end
 
   it 'captures and displays port discovery errors' do
-    helper = described_class.new({ 'FAIL_PORT' => '1' }, ['ssh_mock.rb'])
+    helper = described_class.new({ 'FAIL_PORT' => '1' }, ['ssh'])
     expect { helper.start }
       .to raise_error(/Failed to request.*Something went wrong/m)
   end
 
   it 'captures and displays tunnel errors' do
-    helper = described_class.new({ 'FAIL_TUNNEL' => '1' }, ['ssh_mock.rb'])
+    helper = described_class.new({ 'FAIL_TUNNEL' => '1' }, ['ssh'])
```

Comment:
Out of curiosity, it is not possible to coerce Windows into running the Ruby file as a program directly? Do we *have* to use a wrapper binary here? (I don't mind if we have to, though!)

---

I think the only test that depends on the ssh mock being called `ssh` is in `ps_spec`, but we could realistically stub this out to force it to call the mock instead (like this test does).